scripts/sql/dbsetup.sql

```diff
@@ -6,6 +6,26 @@ BEGIN
 END;
 $$ LANGUAGE plpgsql;
 
+CREATE OR REPLACE FUNCTION add_fhir_endpoint_info_history() RETURNS TRIGGER AS $fhir_endpoints_info_history$
+    BEGIN
+        --
+        -- Create a row in emp_audit to reflect the operation performed on emp,
```

Comment:
"Create a row in fhir_endpoint_info_history to reflect the operation performed on fhir_endpoint"